explainers/wpub-explainer.md

```diff
@@ -136,33 +142,21 @@ The key questions are [1] identifying the "bounds" of the publication, [2] defin
 
 ```
 
-The web publications spec has chosen to use an explicit list of resources. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. The group considered things like `scope` and URL patterns, but felt that an explicit list was simpler. 
-
 ### 2. Sequence of primary resources
 
-- HTML can express ordering of resources via `rel=prev` and `rel=next`, but these do not have UI support from the browsers (with notably rare exceptions).
+Web publications use an explicit list in the manifest to define the reading sequence &#8212; via the `readingOrder` member &#8212; which is again similar to EPUB and its use of a `spine1` element. Other methods considered to achieve sequencing have included:
 
-- HTML can also express an ordered list of links with the `nav` element. 
+- `rel=prev` and `rel=next` in HTML, but these do not have UI support from the browsers (with notably rare exceptions) and are not available for most non-HTML media types.
 
-- Transclusion can also define an ordering of the transcluded components. 
+- identifying sequence via a list of links in a `nav` element (e.g., the table of contents), but not all resources may be listed in the table of contents
 
-- EPUB uses the `spine` element in the XML package file to determine a primary reading order.
-
-Once again, web publications use an explicit list outside of the content itself, via the `readingOrder` member of the manifest. 
+- the DOM order of transcluded documents
 
 ### 3. Metadata
 
-- There are many ways of embedding metadata in individual HTML files
-
-- EPUB includes metadata in the XML package file
-
-
-- Metadata about a web application can be expressed in the web application manifest file. 
-
-- With transclusion, one could define metadata in the parent file to apply to the whole.
-
-Web publication metadata is expressed in the manifest using the schema.org vocabulary, with some extensions. The metadata is serialized as JSON-LD, one of the three syntaxes accepted by schema.org (the others being microdata and RDFa). JSON-LD has also the advantage of being easy to use for both client and server side processing.
+Web publication metadata is expressed in the manifest using the schema.org vocabulary, with some extensions. The metadata is serialized as JSON-LD, one of the three syntaxes accepted by schema.org (the others being microdata and RDFa). JSON-LD has also the advantage of being easy to use for both client and server side processing. As previously, this method is similar to the EPUB approach.
```

Comment:
"As previously, this method is similar to the EPUB approach." I am not sure this is true in this case. There are, obviously, similarities, but the fact of using schema.org and abiding to an external, standard syntax for metadata is, I believe, a significant change compared to EPUB. It is line with the general approach of alignment of publication technologies with what is happening on the Web at large (something worth mentioning, b.t.w.)

Reply:
Sure, but I was just trying to align these subsections with the statement later on that the manifest to date basically represents a translation of the package document to the web. If we make that statement, we should point out how it is true where we discuss these design decisions. I can expand on that sentence, though, to put it in relation to both epub and the web at large.

Reply:
Yeah... saying that it is a 'translation' may be misleading. I would probably prefer 'inspired by', which is somewhat weaker (and, I think, closer to reality). I missed this at the first review cycle...

And adding some words may indeed be helpful...